go.mod

```diff
@@ -1,6 +1,8 @@
 module terraform-provider-genesyscloud
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
```

Comment:
This is happening because of the go version you're using. If you use go1.20 (make sure GOROOT in your Goland settings is pointing to 1.20) then this won't automatically upgrade to 1.21